Library/Formula/hostname.rb

```diff
@@ -0,0 +1,65 @@
+class Hostname < Formula
+  desc "Utility to set/show the host name or domain name"
+  homepage "https://packages.debian.org/sid/hostname"
+  url "http://ftp.de.debian.org/debian/pool/main/h/hostname/hostname_3.15.tar.gz"
+  sha256 "b6d10114ed14306b21745d2cac1b9adf7a2546f16b9fd719bec14bd7cec61d20"
+
+  patch :DATA
```

Comment:
We will not accept it unless it's been submitted (and probably accepted) upstream.